<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-38L647O8" failures="2" tests="8" name="orangeHrmTestcases.TC_002_InvalidLogin" time="9.035" errors="0" timestamp="2022-06-05T18:35:57 EDT" skipped="4">
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="1.110">
    <failure message="expected [Password cannot be empty] but found [Invalid credentials]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Password cannot be empty] but found [Invalid credentials]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at orangeHrmTestcases.TC_002_InvalidLogin.OrangeHrm_TC002_InValidlogin(TC_002_InvalidLogin.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="0.965"/>
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="1.153">
    <skipped/>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="1.663">
    <skipped/>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="1.006">
    <failure message="expected [Username cannot be empty] but found [Invalid credentials]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Username cannot be empty] but found [Invalid credentials]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at orangeHrmTestcases.TC_002_InvalidLogin.OrangeHrm_TC002_InValidlogin(TC_002_InvalidLogin.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="0.996">
    <skipped/>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="0.997">
    <skipped/>
  </testcase> <!-- OrangeHrm_TC002_InValidlogin -->
  <system-out/>
  <testcase classname="orangeHrmTestcases.TC_002_InvalidLogin" name="OrangeHrm_TC002_InValidlogin" time="1.145"/>
  <system-out/>
</testsuite> <!-- orangeHrmTestcases.TC_002_InvalidLogin -->
